### ISSUE TITLE
[Bug]: ut test TestUnlockWithBindNotFound and TestGetLockWithBindNotFound  fail

### ISSUE THREAD

**Issue by YANGGMM** [matrixorigin/matrixone#10962]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="360" alt="image" src="https://github.com/matrixorigin/matrixone/assets/26563383/9f7ee7bd-f2db-4a20-86f5-3ce47fbf66df">
https://pipelines.actions.githubusercontent.com/serviceHosts/420d9c18-f10d-4e16-b77b-a3050f679cb1/_apis/pipelines/1/runs/160110/signedlogcontent/4?urlExpires=2023-08-02T03%3A23%3A30.0610269Z&urlSigningMethod=HMACV1&urlSignature=Iwn18Whi7iYNy%2Bx7CwC3nz8fGv3%2Fzhgl9FKkyN4NCRI%3D

**Comment by zhangxu19830126** [comment]
not working on this issue

**Comment by zhangxu19830126** [comment]
not working on this issue

**Comment by zhangxu19830126** [comment]
not working on this

**Comment by zhangxu19830126** [comment]
not working on this

**Comment by zhangxu19830126** [comment]
not working on this

**Comment by zhangxu19830126** [comment]
fixed by #11148
